lib/event_tracker/tracker.rb

```diff
@@ -6,7 +6,6 @@ def initialize(doer_id, context = {})
       @trackers = []
 
       register_tracker EventTracker::Trackers::Development if development?
-      register_tracker EventTracker::Trackers::Mixpanel if mixpanel?
```

Comment:
Should we also remove a reference to this at RC?

https://github.com/returnly/returns-center/blame/master/app/services/events/README.md#L35

Reply:
yes, thank you! I'll remove it in the opened PR that I have in RC for Mixpanel deprecation phase 1.